networkx/algorithms/operators/binary.py

```diff
@@ -14,7 +14,7 @@
 ]
 
 
-def union(G, H, rename=(None, None), name=None):
```

Comment:
+1 from me for both: removing the unused parameter and doing so in a separate PR with the proper deprecation/future warning!

Reply:
I have added the deprecation warning according to the instructions and other examples I found. Please let me know if I forgot something since it is my first deprecation attempt!